src/content/docs/setup/configuration/cors-setup.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: CORS Setup
+description: Guide to configuring CORS headers for your Adobe Commerce GraphQL endpoints to enable cross-origin requests from your storefront.
+---
+
+import { Aside } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Options from '@components/Options.astro';
+import Option from '@components/Option.astro';
+import Link from '@components/Link.astro';
+
+<Aside type="caution" title="Production recommendation">
+CORS should be a last resort for production. Prefer same-origin delivery of the storefront and API (for example, via Fastly VCL) to avoid CORS entirely.
+</Aside>
+
+## Quick test: Is CORS working?
+
+If you already have CORS configured, test it by replacing `YOUR_STOREFRONT_URL` with your actual frontend URL:
+
+```bash
+curl -I -X OPTIONS https://your-commerce-backend.com/graphql \
```

Comment:
The fact that the `graphql` endpoint is at issue should be explicitly mentioned before the sample command.